### ISSUE TITLE
Update mobile screen "List" and "Map" text color

### ISSUE THREAD

**Issue by sei1122** [hackforla/food-oasis#1674]
### Overview
The mobile screen's **List** and **Map** text color is currently black, which is low color contrast. 

### Action Items
Update to white color to improve accessibility

### Resources/Instructions
![IMG_0643](https://github.com/hackforla/food-oasis/assets/19811032/5c8a4ce1-804b-4114-b617-fd7522286325)
![IMG_0642](https://github.com/hackforla/food-oasis/assets/19811032/417a2970-f858-404b-ab88-caa1df8a8a79)

**Comment by Shubh8899** [comment]
I would like to work on this one.

**Comment by sei1122** [comment]
I talked to @entrotech and sent to dev on May 16

**Comment by 93Belen** [comment]
Right now I can't assign myself to this issue but I am going to be working on it.

**Comment by 93Belen** [comment]
Availability: Tuesday, Thursday and Sunday from 6am to 6pm
ETA: Thursday by 6pm

**Comment by 93Belen** [comment]
Waiting to get access rights to push my changes to the repo.
